lib/network_utils.ml

```diff
@@ -246,21 +250,61 @@ module Sysfs = struct
 			Result.Ok devices.(0)
 		with _ -> Result.Error (Parent_device_of_vf_not_found, "Can not get parent device for " ^ pcibuspath)
 
-	let device_index_of_vf parent_device pcibuspath =
+	let get_child_vfs_sysfs_paths dev = 
```

Comment:
As both functions `device_index_of_vf` and `unbind_child_vfs` need to iterate the VF sysfs paths, I abstract this function `get_child_vfs_sysfs_paths` out from original `device_index_of_vf`